internal/pkg/cli/dialog/create_test_template_test.go

```diff
@@ -0,0 +1,105 @@
+package dialog_test
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/prompt/interactive"
+	"github.com/keboola/keboola-as-code/internal/pkg/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	fixtures "github.com/keboola/keboola-as-code/internal/pkg/fixtures/local"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/template"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/input"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/repository"
+	createOp "github.com/keboola/keboola-as-code/pkg/lib/operation/template/local/test/create"
+)
+
+func TestAskCreateTemplateTestInteractive(t *testing.T) {
+	t.Parallel()
+
+	// Test dependencies
+	dialog, console := createDialogs(t, true)
+	d := dependencies.NewMockedDeps()
+	addMockedObjectsResponses(d.MockedHttpTransport())
+
+	// Set fake file editor
+	dialog.Prompt.(*interactive.Prompt).SetEditor(`true`)
+
+	// Prepare the template
+	fs, err := fixtures.LoadFS("template-simple", env.Empty())
+	assert.NoError(t, err)
+	version, err := model.NewSemVersion("v0.0.1")
+	assert.NoError(t, err)
+	tmplRef := model.NewTemplateRef(model.TemplateRepository{Name: "my-repository"}, "tmpl1", version.String())
+	versionRec := repository.VersionRecord{
+		Version:     version,
+		Description: "",
+		Stable:      true,
+		Components:  []string{},
+		AbsPath:     model.NewAbsPath("", "v1"),
+	}
+	tmplRec := repository.TemplateRecord{
+		Id:          tmplRef.TemplateId(),
+		Name:        "Template 1",
+		Description: "",
+		AbsPath:     model.NewAbsPath("", "tmpl1"),
+		Versions:    []repository.VersionRecord{versionRec},
+	}
+
+	tmpl, err := template.New(tmplRef, tmplRec, versionRec, fs, fs)
+	assert.NoError(t, err)
+
+	// Interaction
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		_, err := console.ExpectString("Api BaseUrl: (https://jsonplaceholder.typicode.com)")
+		assert.NoError(t, err)
+
+		time.Sleep(20 * time.Millisecond)
+		_, err = console.SendLine(`foo.bar.com`)
+		assert.NoError(t, err)
+
+		_, err = console.ExpectString(`Enter the name of the environment variable that will fill input "Api Token". Note that it will get prefix KBC_SECRET_.`)
+		assert.NoError(t, err)
+
+		time.Sleep(20 * time.Millisecond)
+		_, err = console.SendLine(`VAR1`)
+		assert.NoError(t, err)
+
+		_, err = console.ExpectEOF()
+		assert.NoError(t, err)
+	}()
```

Comment:
@webrouse potřeboval bych prosím tě ještě tady poradit proč mi to failuje. To bude zas nejspíš nějaká blbost co pořád přehlížím. 😕 
![CleanShot 2022-09-13 at 18 29 57](https://user-images.githubusercontent.com/215660/189956525-51c5cf17-8a5f-4670-a2e0-77d1683b8dd1.jpg)

Jestli se musí očekávat i ty popisky skupin a kroků, tak to mi taky nefunguje (nebo to je potřeba každý řádek zvlášť?):
![CleanShot 2022-09-13 at 18 36 29](https://user-images.githubusercontent.com/215660/189958019-e2e5e8e2-ee33-4d1e-bd18-33b00e6cc1ca.jpg)
Bohužel nemůžu najít odpovídající příklad v jiných testech. 🙂

Reply:
@JakubMatejka problem bude asi v tom, ze je to v konzole farbne, tj u tam medzi tym aj nejake specialne znaky (?) skus to rozdelit na samostatne `expect` a dat to pod seba. 

Ak by to neslo, tak sa na to pozriem.

Reply:
:eyes:

Reply:
Aha, takže tam neměla být ta default value u tokenu, to mě nenapadlo. 🙂 (Ten otazník a mezeru na konci jsem tam přidával až dneska.) Díky.

Reply:
No ono ta default value je tusim inej farby, ... musime tym interaktivnym testom venovat trocha casu, aby to bolo jednoduchsie....